iOS/FVM/Framework/GestureHandlers/PanGestureHandler.swift

```diff
@@ -53,4 +55,8 @@ extension FVMCarModelViewController {
         heightRatio = heightRatio >= RotateConstraint.maxHeightRatioXUp ? RotateConstraint.maxHeightRatioXUp : heightRatio
         heightRatio = heightRatio <= RotateConstraint.maxHeightRatioXDown ? RotateConstraint.maxHeightRatioXDown : heightRatio
     }
+    
+    fileprivate func hideRotationPrompt() {
+        NotificationCenter.default.post(name: .hideRotationPrompt, object: FVMDamagedCarViewController.self)
```

Comment:
What it's gonna do when rotation prompt is already hidden?

Reply:
I send notification to FVMDamagedCarViewController where is launched function which have to destoy UIImageView - then rotation prompt is hidden